odoo/addons/base/models/ir_model.py

```diff
@@ -116,6 +116,7 @@ def _default_field_id(self):
     @api.depends()
     def _inherited_models(self):
         for model in self:
+            model.inherited_model_ids = []
```

Comment:
The empty list can be confusing, because it would be interpreted as a list of commands instead of a value.
```suggestion
            model.inherited_model_ids = False
```
Better suggestion: assign all records in `self` before the loop
```python
self.inherited_model_ids = False
for model in self:
    ...
```

Reply:
Thanks, done